composer.json

```diff
@@ -32,7 +32,8 @@
     },
     "require-dev": {
         "phpunit/phpunit" : "4.*",
-        "scrutinizer/ocular": "~1.1"
+        "scrutinizer/ocular": "~1.1",
```

Comment:
If this mob uses semantic version, why not ~1 (i.e. anyof the 1 series)? OTOH their versioning could be as machine readable as the Laravel Framework's.

Reply:
Hmm... that sounds dangerous. Sometimes small breaking changes are introduced by point updates... I know that because both Laravel and we do that too :-)